package.json

```diff
@@ -19,7 +19,7 @@
     "test": "turbo run test --output-logs=new-only --concurrency=1",
     "test:match": "cd packages/astro && pnpm run test:match",
     "test:templates": "turbo run test --filter=create-astro --concurrency=1",
-    "test:smoke": "node scripts/smoke/index.js",
+    "test:smoke": "turbo run build --filter=\"@example/*\" --filter=\"!@example/with-mdx\" --output-logs=new-only",
```

Comment:
Temp: exclude `@example/with-mdx` as we investigate hanging production builds. Looks like the `client:` directive introduces the issue.